src/main/java/org/springframework/data/couchbase/repository/query/StringN1qlQueryCreator.java

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright 2012-2020 the original author or authors
```

Comment:
Nit: The from date is typically reflecting the inception year. So `Copyright 2020 the …` is fine.

Reply:
ok. I fixed this one.